### ISSUE TITLE
MOKO resources are not copied into build/js folder

### ISSUE THREAD

**Issue by ComBatVision** [WorldWindEarth/WorldWindKotlin#1]
The Android version of example application builds fine and see resources from "worldwind" common resources, but the JS example application shows an error during build, because it does not see images in the generated build/js folder. All dependent JS libraries are there, but no resources.

I assume that it is a MOKO resources library Gradle plugin issue when resources are in the dependent module of the same project and it does not generate/copy resources to the output folder of the application.

Issue was reported to MOKO developers https://github.com/icerockdev/moko-resources/issues/385

JS examples cannot be merged to develop until this issue fixed. 
To reproduce an issue look at branch https://github.com/WorldWindEarth/WorldWindKotlin/tree/feature/worldwind-examples-js

![error](https://user-images.githubusercontent.com/12487306/191712436-28cf6c5a-c796-4ae6-9fc7-01570c8d2eaa.png)
![files](https://user-images.githubusercontent.com/12487306/191712466-787fe315-2dfc-400d-8d72-fdd01df7bf86.png)

**Comment by ComBatVision** [comment]
As a temporary workaround to run JS example the following code in generated  WorldWindKotin-worldwind-js.js file:
```
  function images() {
    images_instance = this;
    var tmp = this;
    var tmp_0 = require('images/dnb_land_ocean_ice_2012.png');
    var tmp0_fileUrl = (!(tmp_0 == null) ? typeof tmp_0 === 'string' : false) ? tmp_0 : THROW_CCE();
    tmp.dnb_land_ocean_ice_2012__1 = new ImageResource('dnb_land_ocean_ice_2012.png', tmp0_fileUrl);
    var tmp_1 = this;
    var tmp_2 = require('images/worldwind_worldtopobathy2004053.png');
    var tmp0_fileUrl_0 = (!(tmp_2 == null) ? typeof tmp_2 === 'string' : false) ? tmp_2 : THROW_CCE();
    tmp_1.worldwind_worldtopobathy2004053__1 = new ImageResource('worldwind_worldtopobathy2004053.png', tmp0_fileUrl_0);
  }
```
must be replaced with the following code, respecting trail empty lines to keep consistency between Kotlin and JS in debugger:
```
  function images() {
    images_instance = this;
    var tmp = this;
    tmp.dnb_land_ocean_ice_2012__1 = new ImageResource('dnb_land_ocean_ice_2012.png', './images/dnb_land_ocean_ice_2012.png');
    var tmp_1 = this;
    tmp_1.worldwind_worldtopobathy2004053__1 = new ImageResource('worldwind_worldtopobathy2004053.png', './images/worldwind_worldtopobathy2004053.png');
  }




```

**Comment by ComBatVision** [comment]
Fixed in 65c1c19af28671e28fb6ba40c80e606195181c7c
